src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -108,6 +111,38 @@ class VAMap extends Component {
       resultsPage = 1;
     }
 
+    /*
+      Notes:
+
+      Going to need a couple new flags in the Redux store to properly
+      track state of the app. For example, a flag to know when Mapbox API
+      requests are done as all they do is update the Redux store, but intuiting
+      whether or not the data in the fields that were updated represents a valid
+      state for triggering a new search is ambiguous at best nor should we simply
+      fire off a new search each time something changes in Redux.
+
+      New Flag Ideas:
+        - geocodeInProgress
+        - revGeocodeInProgress - should be a separate flag as both operations happen
+        - searchRequested - To track that the user clicked the search button
+          (could have used inProgress but it gets tripped by other Actions)
+        - 
+      
+      The boundary checking of the current code below doesn't actually work.
+      Array equality isn't something that should be done with the operator,
+      and using the new method below causes `searchWithBounds` to never fire.
+      Goes in line with needing clearer ideas of what state of the app ==
+      when to fire off a new search, zoom out, or even just do nothing.
+
+      Near as I can tell this.zoomOut.cancel() does nothing.
```

Comment:
Should the points in these notes be made tickets instead? It's probably totally appropriate to leave comments pointing to the tickets, but I think these won't be addressed until they're put into tickets to be prioritized.

Reply:
Yes, there should probably be some bugs/tickets opened for some of the behavior of the app, although I've been having trouble isolating exactly where it's coming from so I've got these notes in for now + that linked gist with my notes on a debugging session.
Any help with it would be appreciated. Would it be faster to open new bugs to do that?